docs/data/material/getting-started/templates/sign-in/SignIn.tsx

```diff
@@ -39,73 +111,266 @@ export default function SignIn() {
+              }}
+            >
+              <InputLabel htmlFor="email" sx={visuallyHidden}>
+                Email
+              </InputLabel>
+              <TextField
+                error={emailError}
+                helperText={emailErrorMessage}
+                id="email"
+                type="email"
+                name="email"
+                placeholder="Email address"
+                autoComplete="email"
+                autoFocus
+                required
+                fullWidth
+                variant="outlined"
+                color={emailError ? 'error' : 'primary'}
+                sx={{ ariaLabel: 'email' }}
+              />
+              <InputLabel htmlFor="password" sx={visuallyHidden}>
+                Password
+              </InputLabel>
+              <TextField
+                error={passwordError}
+                helperText={passwordErrorMessage}
+                name="password"
+                placeholder="Password"
+                type="password"
+                id="password"
+                autoComplete="current-password"
+                autoFocus
+                required
+                fullWidth
+                variant="outlined"
+                color={passwordError ? 'error' : 'primary'}
+              />
+              <Box
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'space-between',
+                  width: '100%',
+                }}
+              >
+                <FormControlLabel
+                  control={<Checkbox value="remember" color="primary" />}
+                  label="Remember me"
+                />
+                <Link component="button" onClick={handleClickOpen} variant="body2">
+                  Forgot your password?
                 </Link>
-              </Grid>
-            </Grid>
-          </Box>
-        </Box>
-        <Copyright sx={{ mt: 8, mb: 4 }} />
-      </Container>
+              </Box>
+              <ForgotPassword open={open} handleClose={handleClose} />
+              <Button
+                type="submit"
+                fullWidth
+                variant="contained"
+                onClick={validateInputs}
+              >
+                Sign in
+              </Button>
+              <Link
+                href="/material-ui/getting-started/templates/sign-up/"
+                variant="body2"
+                sx={{ alignSelf: 'center' }}
+              >
+                Don&apos;t have an account? Sign up
+              </Link>
+            </Box>
+            <Divider>or</Divider>
+            <Box sx={{ display: 'flex', flexDirection: 'column', gap: 2 }}>
+              <Button
+                type="submit"
+                fullWidth
+                variant="outlined"
+                color="secondary"
+                onClick={() => alert('Sign in with Google')}
+                startIcon={
+                  <SvgIcon>
```

Comment:
I'd create these SvgIcons outside of the main component and just import them here as `<GoogleIcon/>` or something. We could even think of doing it in a separate file altogether.